index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./assets/style/index.css">
+	<title>Tourism Website</title>
+</head>
+
+<body>
+	<header>
+
+		<nav class="menu">
+			<div class="burger">
+				<div class="line1"></div>
+				<div class="line2"></div>
+				<div class="line3"></div>
+			</div>
+		</nav>
+		<ul class="nav-ul">
+			<li><a href="./index.html">
+					<img class="logo-img" src="./assets/images/logo.svg" alt=""></a></li>
+			<li class="navbar"><a class="nav-link" href="./about.html">About</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_2">Benefits</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_3">Tours</a></li>
+			<li class="navbar"><a class="nav-link" href="#section_4">Contact</a></li>
+		</ul>
+	</header>
+	<section class="section_1" id="section_1">
+		<h2 class="main-text">
+			Your Perfect Tour in Uzbekistan Starts Here!
+		</h2>
+		<p class="text">
+			If you are planning a trip to Uzbekistan, Lorem ipsum is the best choice.Just specify the region you
+			prefer
+			to visit and one of our expert travel guides will provide you with a well-thought-out vacation plan. Book
+			your trip and enjoy the amazing experience!
+		</p>
+	</section>
+
+	<section class="section_2" id="section_2">
+		<h2 class="text-2">Main benefits</h2>
+		<div class="line-separator"></div>
+		<div class="section2-box">
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/map_white_48dp.svg" alt="">
+				<h5>What we offer</h5>
+				<p class="box-text">We offer hundreds of sightseeing tours and custom packages which can meet any
+					taste
+					and budget.
+				</p>
+			</div>
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/verified_white_48dp.svg" alt="">
+				<h5>Our experience</h5>
+				<p class="box-text">Each of our travel specialists meets first-class industry standards and has at
+					least
+					4.5 out of 5-star
+					rating.
+				</p>
+			</div>
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/thumb_up_alt_white_48dp.svg" alt="">
+				<h5>Satisfaction guaranteed</h5>
+				<p class="box-text">Our experienced travel agents guarantee each client the most enjoyable and
+					memorable
+					experience.
+				</p>
+			</div>
+		</div>
+		<button type="button" class="btn">Book a tour</button>
+		<h2 class="all-benefits">SEE THE WHOLE BENEFITS</h2>
+	</section>
+
```

Comment:
- [ ] Let us add more boxes like 4 or 5 to meet the design requirements.

Reply:
This issue is well resolved! ✔️